app/js/lib/validation_dispatchers/base.js

```diff
@@ -20,6 +20,7 @@ var _ = require( 'underscore' ),
 		finishActionCreationFunction: null,
 		beginActionCreationFunction: null,
 		fields: null,
+		fieldValuesIfNotSet: null,
```

Comment:
I'm not sure about the name of this. It could be accurate, but I'm unable to figure out what exactly this is. Is it the initial values? Looking at the loop below that does not seem to be far off, but then the derivatives are not specifying this for all their fields...

Perhaps a description that says this should be a map with the fields required for dispatch and their initial values?